src/components/Landing.js

```diff
@@ -1,21 +1,99 @@
 import React, {useState} from "react";
+import styled from "styled-components";
+import {background} from "../img/index";
+// import {LoginContext} from "../context/LoginContext";
 
 const Landing = () => {
+  // const {isAuthenticated, handleSubmit, handleChange} = useContext(LoginContext);
   const [password, setPassword] = useState("");
+  // check in localstorage before setting this value to false
+  const [isAuthenticated, setIsAuthenticated] = useState(false);
+
+  const handleChange = (e) => {
+    const input = e.target.value;
+    console.log("input: ", input);
+    setPassword(input);
+  };
 
   const handleSubmit = (e) => {
     e.preventDefault();
+    console.log("Submitted");
+    if (password !== "123") {
```

Comment:
Probably want this to be configurable because it's used in more than one place. We want some guarantee that they will always be the same